applications/CoSimulationApplication/README.md

```diff
@@ -238,7 +238,7 @@ The _CoSimulationApplication_ consists of the following main components (taken f
 - **IO**: Responsible for communicating and data exchange with external solvers/codes
 - **DataTransferOperator**: Transfers data from one discretization to another, e.g. by use of mapping techniques
 - **CouplingOperation**: Tool for customizing coupled simulations
-- **ConvergenceAccelerator**: Accelerating the solution in strongly coupled simulations by use of relaxation techniques
+- **ConvergenceAccelerator**: Accelerating the solution in strongly coupled simulations by use of Jacobian-free or Quasi-Newton techniques
```

Comment:
```suggestion
- **ConvergenceAccelerator**: Accelerating the solution in strongly coupled simulations by use of relaxation or Quasi-Newton techniques
```
As far as I know there are no Jacobian-free techniques implemented. However, there is an Aitken relaxation convergence accelerator.

Reply:
@pooyan-dadvand this is what I suggested